Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -1,116 +1,169 @@
+        4.  creating sub-committees or working groups to focus on cross-project technical issues and requirements;
 
-   2. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+        5.  appointing representatives to work with other open source or open standards communities;
 
-   3. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
+        6.  establishing community norms, workflows, issuing releases, and security issue reporting policies;
 
-   4. In the event a vote cannot be resolved by the TSC, any voting member of the TSC may refer the matter to the Series Manager for assistance in reaching a resolution.
+        7.  approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 8 of this Charter;
 
-4. **Compliance with Policies** 
+        8.  discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
 
-   1. This Charter is subject to the Series Agreement for the Project and the Operating Agreement of LF Projects. Contributors will comply with the policies of LF Projects as may be adopted and amended by LF Projects, including, without limitation the policies listed at https://lfprojects.org/policies/.  
+        9.  coordinating any marketing, events, or communications regarding the Project.
 
-   2. The TSC may adopt a code of conduct (“CoC”) for the Project, which is subject to approval by the Series Manager.  In the event that a Project-specific CoC has not been approved, the LF Projects Code of Conduct listed at [https://lfprojects.org/policies](https://lfprojects.org/policies) will apply for all Collaborators in the Project.
+4.  **Sub-Project Maintainers**
 
-   3. When amending or adopting any policy applicable to the Project, LF Projects will publish such policy, as to be amended or adopted, on its web site at least 30 days prior to such policy taking effect; provided, however, that in the case of any amendment of the Trademark Policy or Terms of Use of LF Projects, any such amendment is effective upon publication on LF Project’s web site.
+    1.  **Definition:** Each officially recognized **Sub-Project** within the ecosystem shall have its own group of **Sub-Project Maintainers** (hereinafter "Maintainers"). These Maintainers are the individuals who can be members of the TSC as per Section 3.2.
```

Comment:
The focus on sub-projects loses the fact that repositories within a sub-project can have different sets of maintainers.

Reply:
I have updated the TSC formation to address this issue.